src/main/java/ca/corefacility/bioinformatics/irida/ria/web/announcements/dto/AnnouncementTableModel.java

```diff
@@ -0,0 +1,35 @@
+package ca.corefacility.bioinformatics.irida.ria.web.announcements.dto;
+
+import ca.corefacility.bioinformatics.irida.model.announcements.Announcement;
+import ca.corefacility.bioinformatics.irida.model.user.User;
+import ca.corefacility.bioinformatics.irida.ria.web.models.tables.TableModel;
+
+/**
+ * User interface model for DataTables for administration of {@link Announcement}
```

Comment:
Update this comment as it is no longer using datatables

Reply:
Updated in e78dce8f8120639b24b6d31d972e7659978471da